R/utils.R

```diff
@@ -20,9 +20,15 @@ auto_name <- function(names) {
   names
 }
 
-# Dummy definition of tibble::type_sum to avoid R CMD CHECK warning
-type_sum <- function(x) UseMethod("type_sum")
-
 is_windows <- function() {
   tolower(Sys.info()[["sysname"]]) == "windows"
 }
+
+# This is needed to avoid checking the class of fs_filename objects in the
+# tests.
+compare.fs_filename <- function(x, y) {
+  if (identical(class(y), "character")) {
+    class(x) <- "character"
```

Comment:
Setting the class of a base type is a slightly odd thing to do.

Reply:
Yeah you are right, I can just use `class(x) <- NULL` here and get the same result.